docs/feature-flags/sdks/node.md

```diff
@@ -101,21 +101,32 @@ const variation = eppoClient.getAssignment(
 );
 ```
 
-The `getAssignment` function takes two required and one optional input to assign a variation:
+The `getStringAssignment` function takes two required and one optional input to assign a variation:
 
 - `subjectKey` - The entity ID that is being experimented on, typically represented by a uuid.
 - `flagOrExperimentKey` - This key is available on the detail page for both flags and experiments.
-- `targetingAttributes` - An optional map of metadata about the subject used for targeting. If you create rules based on attributes on a flag/experiment, those attributes should be passed in on every assignment call.
+- `subjectAttributes` - An optional map of metadata about the subject used for targeting. If you create rules based on attributes on a flag/experiment, those attributes should be passed in on every assignment call.
+
+### Typed assignments
+
+Additional functions are available:
+
+```
+getBoolAssignment(...)
```

Comment:
Sometimes "Bool", sometimes "Boolean" 😑

Reply:
Is it too late to rename functions?

Reply:
We can add the `getBooleanAssignment(...)` function everywhere and add a deprecation warning to the `getBoolAssignment` for example, and remove it in a future version of the SDK

Reply:
Yeah given the type in JavaScript is "boolean" this is probably the (low priority) path we should take.

Reply:
Added to Linear. This would be nice to do, perhaps we can wrap this into the next round of SDK updates

https://linear.app/eppo/issue/FF-1002/getbool-to-getboolean